Documentation/ExtensionInstallation/Index.rst

```diff
@@ -55,10 +63,73 @@ In the backend:
 8. Click on the icon with a :guilabel:`"+"` sign for your extension
    in the :guilabel:`"A/D"` column.
 
+Uninstall an Extension Without Composer
+=======================================
+
+If you installed TYPO3 via composer you should uninstall Extensions via composer.
+
+Check Dependencies
+------------------
+
+First find out, which other extensions and functions of your Typo3-Installation are dependent of the extension you
+want to deinstall. Major dependeancies you can find out by checking the Extension Repository. Look for the extension
+you want to deinstall and the others you have installed. Read for all of them the chapters 'Dependencies' and
+'Reverse dependencies'.
+
+Contemplate, whether you made referrals to the extension in any setup, config or other Typoscript-section. Think of
+the results of deleting them and finally do it.
+
+If you are not sure, or just testing, you might as well deinstall the extension (which is without deleting) and see
```

Comment:
This may be practical advice, but for an enterprise CMS, this does not sound good, IMHO. Personally, I never "just see what happens" on my live production system. I always have development or staging systems where I do things. Of course, even this does not prevent all errors from happening.

So I wouldn't remove the part about the PackageStates.php because I think that is good to know how you can fix things you can't fix from the BE. But I would maybe phrase it differently. 

Maybe just remove "If you are not sure .." sentence and write something like "If you get an exception and can't even access the EM anymore because of it, you can deactivate / activate extensions manually with PackageStates.php as a last resort, see `<add link here>`"